src/explorer/trialApp/TrialAppTreeItem.ts

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzExtTreeItem, IActionContext } from 'vscode-azureextensionui';
+import { localize } from '../../localize';
+import { openUrl } from '../../utils/openUrl';
+import { requestUtils } from '../../utils/requestUtils';
+import { AzureAccountTreeItem } from '../AzureAccountTreeItem';
+import { ISiteTreeItem } from '../ISiteTreeItem';
+import { SiteTreeItemBase } from '../SiteTreeItemBase';
+import { ITrialAppMetadata } from './ITrialAppMetadata';
+
+export class TrialAppTreeItem extends SiteTreeItemBase implements ISiteTreeItem {
+    public static contextValue: string = 'trialApp';
+    public contextValue: string = TrialAppTreeItem.contextValue;
+
```

Comment:
nit: Could you reduce the number of empty lines?  Just leave what is currently line 24, seperating the pubic and private properties.